src/Components/IncidentList/IncidentCard.Component.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class IncidentCard extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render () {
+    return (
+      <div className="incident-card">
+        <div className="incident-header">
+          <span className="incident-subject">
+            {this.props.incidentSubject}
```

Comment:
Use de-structuring to make you code more concise

Reply:
done